platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/Installer.java

```diff
@@ -62,4 +67,17 @@ public void validate() throws IllegalStateException {
         System.setProperty( "flatlaf.updateUIOnSystemFontChange", "false" );
     }
 
+    private void uiPropertyChange(PropertyChangeEvent pce) {
+        if ("lookAndFeel".equals(pce.getPropertyName())) { //NOI18N
+            if (UIManager.getLookAndFeel() instanceof FlatLaf
+                    && FlatLafPrefs.isUseWindowDecorations()
+                    && SystemInfo.isLinux) {
+
+                JFrame.setDefaultLookAndFeelDecorated(true);
+                JDialog.setDefaultLookAndFeelDecorated(true);
```

Comment:
> If there is a listener that turns these on, should there not be some code turning them off again? Is this code only applicable when the setting or LAF is changed while NetBeans is running? Or does it end up running on every startup?

It runs only on startup because changing LAF needs a NetBeans restart.

Reply:
OK. Is the Installer the best place to put the setDefaultLookAndFeelDecorated calls? Are we guaranteed to get a PropertyChangeEvent when the LAF is initially set?

A possibly cleaner alternative would be to add a dedicated windowDefaultLookAndFeelDecorated property that can be returned from LFCustoms.createApplicationSpecificKeysAndValues, and which then triggers the call to JFrame/JDialog.setDefaultLookAndFeelDecorated from o.n.swing.plaf.Startup.installLFCustoms. (I'd only bother with this if others think it's a good idea.)

Reply:
Or look at using `nb.laf.postinstall.callable` ? See https://github.com/apache/netbeans/blob/master/platform/o.n.swing.plaf/src/org/netbeans/swing/plaf/Startup.java#L339  It might run too late though?

Reply:
@neilcsmith-net I tried `nb.laf.postinstall.callable` and does not work. Yeah, SwingUtilities.invokeLater run too late.

@eirikbakke I did. Can you take a look?